storage/sql/config.go

```diff
@@ -88,6 +89,12 @@ type Postgres struct {
 	SSL PostgresSSL `json:"ssl" yaml:"ssl"`
 
 	ConnectionTimeout int // Seconds
+
+	// database/sql tunables, see
```

Comment:
When would it be appropriate to set these to a higher value? (about how many clients?) What's the default?

Reply:
The defaults right now should be zeros, which is the same as the stdlib default.

Reply:
Realised I hadn't answered your question _at all_ there, but it was time to go to bed. I'll update the _issue_ #1354 with some details, as I'd think that's what people might find first.

Reply:
Can we include defaults here that we feel are reasonable values?

Reply:
👍 sure. I'm not sure what makes for good defaults, but I can try 😅